lang/cs/Org.Apache.REEF.IO/FileSystem/IFileSystem.cs

```diff
@@ -104,5 +104,11 @@ public interface IFileSystem
         /// <returns></returns>
         /// <exception cref="IOException"></exception>
         IEnumerable<Uri> GetChildren(Uri directoryUri);
+
+        /// <summary>
+        /// Returns Url prefix 
+        /// </summary>
+        /// <returns></returns>
+        string UriPrefix { get; }
```

Comment:
well, if we can wrap it inside HadoopFileSystem copy methods, then we don't need to expose it. Currently in Partition class, we get IFileSystem, we need to create a proper Uri then call copy method through the interface. We don't want to downcast IFileSystem apparently.